lib/src/templates/very_good_core/very_good_core_template.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:io/ansi.dart';
+import 'package:mason/mason.dart';
+import 'package:universal_io/io.dart';
+import 'package:very_good_cli/src/templates/templates.dart';
+
+/// {@template core_template}
+/// A core Flutter app template.
+/// {@endtemplate}
+class CoreTemplate extends Template {
```

Comment:
```suggestion
class VeryGoodCoreTemplate extends Template {
```

thoughts on renaming to VeryGoodCoreTemplate for consistency with the file name?

Reply:
Also, I think this renaming will imply to renaming docs

Reply:
yup good call! I'll make the change shortly 👍